data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditMessage.java

```diff
@@ -97,18 +114,31 @@ protected AuditMessage(
     }
 
     @Generated
+    public String getLeafResourceId() {
+        return leafResourceId;
+    }
+
+    @Generated
+    public String getToken() {
+        return token;
+    }
+
+    @Generated
+    @Nullable
     public String getUserId() {
         return userId;
```

Comment:
```suggestion
    public Optional<String> getUserId() {
        return Optional.ofNullable(userId);
```

Reply:
Will raise as ticked to cover this across all of the services